site/content/en/docs/Getting started/first_kustomization.md

```diff
@@ -4,48 +4,98 @@ linkTitle: "Creating Your First Kustomization"
 date: 2022-02-27
 weight: 20
 description: >
-  A simple project example to get you familiar with the concepts
+  A step by step tutorial for absolute kustomize beginners
 ---
 
-We're going to use kustomize to deploy an nginx instance into our Kubernetes cluster.
+This page is intended to help you get started with using this amazing tool called kustomize! We will start off with creating a simple nginx deployment manifest and then use it as an example to explore kustomize basics.
```

Comment:
Do we have a style guide on how we want to write the word kustomize consistently?
* Kustomize?
* kustomize?
* `kustomize`?

I'm a fan of the last one but things can be a bit cluttered sometimes when you have too many words in backticks flying around.

Reply:
I prefer the second one, the word kustomize will be used a lot and back ticking every single one will start to decrease its emphasis and/or may get annoying to look at.